docs/_static/custom.css

```diff
@@ -1 +1,124 @@
-#version-picker-label { color: white; display: inline; }
+@import url('https://fonts.googleapis.com/css?family=Space+Mono&display=swap');
```

Comment:
Can you format this file?
There are lots of unneeded empty lines and some of the indentation is off. There is probably a tool like `prettier` that can help!